src/ingestor.ts

```diff
@@ -79,6 +80,16 @@ export default async function ingestor(req) {
     if (!Object.keys(hashTypes).includes(hash)) return Promise.reject('wrong types');
     type = hashTypes[hash];
 
+    try {
+      if (
+        (message.space.startsWith('s:') || !message.space.includes(':')) &&
+        ensNormalize(message.space) !== message.space.toLowerCase()
```

Comment:
Maybe we can move to `getSpace`?

Reply:
See https://github.com/snapshot-labs/snapshot-sequencer/pull/417#issuecomment-2255396997

Reply:
lolol